content/docs/hello-world.md

```diff
@@ -1,50 +1,50 @@
 ---
 id: hello-world
-title: Hello World
+title: नमस्ते दुनिया
 permalink: docs/hello-world.html
 prev: cdn-links.html
 next: introducing-jsx.html
 ---
 
-The smallest React example looks like this:
+React का सबसे छोटा उदाहरण इस प्रकार है:
 
 ```js
 ReactDOM.render(
-  <h1>Hello, world!</h1>,
+  <h1>नमस्ते, दुनिया!</h1>,
   document.getElementById('root')
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+यह पृष्ठ पर "नमस्ते, दुनिया!" कहते हुए शीर्षक को दिखाए गा।
 
 [](codepen://hello-world)
 
-Click the link above to open an online editor. Feel free to make some changes, and see how they affect the output. Most pages in this guide will have editable examples like this one.
+ऑनलाइन संपादक खोलने के लिए ऊपर दिए गए लिंक पर क्लिक करें। कुछ बदलाव करने के लिए स्वतंत्र महसूस करें, और देखें कि वे आउटपुट को कैसे प्रभावित करते हैं। इस गाइड के अधिकांश पृष्ठों में इस तरह के संपादन योग्य उदाहरण होंगे।
 
 
-## How to Read This Guide {#how-to-read-this-guide}
+## इस गाइड को कैसे पढ़ें {#how-to-read-this-guide}
 
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+इस गाइड में, हम React एप्लिकेशन के बिल्डिंग ब्लॉक्स: एलेमेन्त्स्  और कौम्पोनॅन्ट्स की जांच करेंगे। एक बार जब आप इसमे महारत हासिल कर लेते हैं, तो आप छोटे पुन: प्रयोज्य टुकड़ों से जटिल एप्लिकेशन बना सकते हैं।
 
->Tip
+>टिप
 >
->This guide is designed for people who prefer **learning concepts step by step**. If you prefer to learn by doing, check out our [practical tutorial](/tutorial/tutorial.html). You might find this guide and the tutorial complementary to each other.
+>यह मार्गदर्शिका ऐसे लोगों के लिए डिज़ाइन की गई है जो **कॉन्सेप्ट्स को चरण दर चरण** सीखना पसंद करते हैं। यदि आप ऐसा करके सीखना पसंद करते हैं, तो हमारे [व्यावहारिक ट्यूटोरियल](/tutorial/tutorial.html) को देखें। आपको यह मार्गदर्शिका और ट्यूटोरियल एक-दूसरे के पूरक लग सकते हैं।
```

Comment:
चरण दर चरण is the literal translation but, something better can be put in I think

Reply:
व्यावहारिक  can be replaced by hindi word practical